### PR TITLE
refactor: drop some modules in `@taiga-ui/addon-mobile`

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by mdlufy — APPROVED** [review]

File: projects/addon-doc/utils/test/try-parse-markdown.spec.ts [hunk 1]
```diff
@@ -42,10 +42,10 @@ ${codeSection}html\n${htmlCode}\n${codeSection}
 @NgModule({
     imports: [
         // ...
-        TuiAvatarModule,
+        TuiAvatar,
```
Comment (waterplea) [inline projects/addon-doc/utils/test/try-parse-markdown.spec.ts:45]: Accidental change?

File: projects/addon-mobile/components/index.ts [hunk 2]
```diff
@@ -3,7 +3,6 @@ export * from '@taiga-ui/addon-mobile/components/mobile-calendar-dialog';
 export * from '@taiga-ui/addon-mobile/components/mobile-calendar-dropdown';
 export * from '@taiga-ui/addon-mobile/components/mobile-dialog';
 export * from '@taiga-ui/addon-mobile/components/pull-to-refresh';
-export * from '@taiga-ui/addon-mobile/components/sheet';
```
Comment (waterplea) [inline projects/addon-mobile/components/index.ts:6]: We should move it to legacy, there are some features we cannot replace at the moment.

**Review by waterplea — APPROVED** [review]

File: projects/addon-doc/utils/test/try-parse-markdown.spec.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-doc/utils/test/try-parse-markdown.spec.ts:45]: It's a just spec file. I just didn't want to have any mention of the module

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 3]
```diff
@@ -1244,4 +1244,70 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
...
+        },
+        to: {
+            name: 'TuiTouchableDirective',
+            moduleSpecifier: '@taiga-ui/addon-mobile',
+        },
+    },
+    {
+        from: {
+            name: 'TuiElasticStickyModule',
+            moduleSpecifier: '@taiga-ui/addon-commerce',
```
Comment (vladimirpotekhin) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:1280]: from `addon-mobile` ? (and below and above)

Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts]: fixed

File: projects/addon-mobile/components/index.ts (hunk 2 above)
Comment (splincode) [reply projects/addon-mobile/components/index.ts:6]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**